src/test/java/core/basesyntax/service/impl/ReportCreatorServiceImplTest.java

```diff
@@ -0,0 +1,39 @@
+package core.basesyntax.service.impl;
+
+import static org.junit.Assert.assertEquals;
+import static org.junit.Assert.assertNotEquals;
+
+import core.basesyntax.db.Storage;
+import core.basesyntax.service.ReportCreatorService;
+import org.junit.After;
+import org.junit.BeforeClass;
+import org.junit.Test;
+
+public class ReportCreatorServiceImplTest {
+    private static ReportCreatorService reportCreatorService = new ReportCreatorServiceImpl();
+    private static String expected;
+
+    @BeforeClass
+    public static void beforeClass() {
+        expected = "fruit,quantity" + System.lineSeparator()
+                + "banana,100" + System.lineSeparator();
+    }
+
+    @Test
+    public void createReport_validReport_ok() {
+        Storage.fruits.put("banana", 100);
+        String actual = reportCreatorService.createReport();
+        assertEquals(expected, actual);
+    }
+
+    @Test
+    public void createReport_emptyStorage_notOk() {

```

Comment:
I guess the name "notOk" is used if Exception is expected. But it is not that case here.

Reply:
I think that we can use "notOk" in any situation that we consider invalid and not only when Exception is expected, but it may be wrong, I hope mentor will help us with this question)